logstash-input-http_poller.gemspec

```diff
@@ -17,13 +17,12 @@ Gem::Specification.new do |s|
   s.metadata = { "logstash_plugin" => "true", "logstash_group" => "input" }
 
   # Gem dependencies
-  s.add_runtime_dependency "logstash-core", ">= 2.0.0", "< 3.0.0"
+  s.add_runtime_dependency "logstash-core", "2.0.0.snapshot2"
```

Comment:
I was debating doing this as well. I switched this for testing, but left it as before when pushing code...

how should we keep this? `snapshot`?